pkg/graph/resolvers/trb_request_attendee_test.go

```diff
@@ -0,0 +1,45 @@
+package resolvers
+
+import (
+	"context"
+
+	"github.com/cmsgov/easi-app/pkg/models"
+)
+
+// TestCreateTRBRequestAttendee makes a new TRB request
+func (s *ResolverSuite) TestCreateTRBRequestAttendee() {
+	ctx := context.Background()
+	anonEua := "ANON"
+	trbRequest := models.NewTRBRequest(anonEua)
+	trbRequest.Type = models.TRBTNeedHelp
+	trbRequest.Status = models.TRBSOpen
+	trbRequest, err := CreateTRBRequest(s.testConfigs.Context, models.TRBTBrainstorm, s.testConfigs.Store)
+	s.NoError(err)
+
+	s.Run("create a TRB request attendee", func() {
+		attendee := models.TRBRequestAttendee{
+			EUAUserID:    anonEua,
+			TRBRequestID: trbRequest.ID,
+			Role:         models.PersonRolePrivacyAdvisor,
+		}
+		attendee.CreatedBy = anonEua
+		createdAttendee, err := s.testConfigs.Store.CreateTRBRequestAttendee(ctx, &attendee)
+		s.NoError(err)
+
+		createdAttendee.Role = models.PersonRoleCloudNavigator
+		createdAttendee.ModifiedBy = &anonEua
+		_, err = s.testConfigs.Store.UpdateTRBRequestAttendee(ctx, createdAttendee)
+		s.NoError(err)
+
+		createdAttendee.Component = "The Citadel of Ricks"
+		createdAttendee.ModifiedBy = &anonEua
+		_, err = s.testConfigs.Store.UpdateTRBRequestAttendee(ctx, createdAttendee)
+		s.NoError(err)
+	})
+
+	s.Run("fetches TRB request attendees", func() {
+		fetched, err := s.testConfigs.Store.GetTRBRequestAttendeesByTRBRequestID(ctx, trbRequest.ID)
+		s.NoError(err)
+		s.True(len(fetched) > 0)
+	})
```

Comment:
What functions are under test here? This being in the `resolvers` package would make me think we'd be testing resolver functions, but I only seem to se references to `store` methods in here

I think we should probably be calling `TRBRequestCreate`, `TRBRequestUpdate`, and `TRBRequestCollectionGet` and asserting that the return values are what we expect

Reply:
@ClayBenson94 How does it look now?

Reply:
It looks good! 🎉 

Initially, I was a bit confused as to how the same TRB request is being modified/accessed between multiple tests ([since we reset the DB between each test](https://github.com/CMSgov/easi-app/blob/e4fdd89380c97b30865a1823449ed187911f8129/pkg/graph/resolvers/resolver_test.go#L27-L31)) but it looks like `s.Run` might get around that, so that's why this is able to work! Don't think this needs to change now, but is worth thinking about